.gitattributes

```diff
@@ -1,4 +1,6 @@
 * text eol=lf
 
+*.png binary
```

Comment:
What about JPG?

Reply:
I don't think Git will treat JPG as text because of its binary format

Reply:
> I don't think Git will treat JPG as text because of its binary format

Anyway

```
The file will have its original line endings in your working directory
warning: CRLF will be replaced by LF in images/internal_flow.jpg.
```

And

```
❯ git status
On branch al-broken-pngs
Your branch is up to date with 'origin/al-broken-pngs'.

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	modified:   images/internal_flow.jpg

no changes added to commit (use "git add" and/or "git commit -a")
```